core-ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -141,7 +141,7 @@ fun NewsResourceHeaderImage(
         },
         modifier = Modifier
             .fillMaxWidth()
-            .clip(RoundedCornerShape(topEnd = 24.dp, topStart = 24.dp))
+            .clip(RoundedCornerShape(topEnd = 16.dp, topStart = 16.dp))
```

Comment:
what do you think about extracting this value out as a variable (either in this file directly, or in dimens.xml) so that it's less error prone if we decide to update it later?

Reply:
Clip is no longer necessary on the child element.